lib/libpcsc-cpp/include/pcsc-cpp/pcsc-cpp.hpp

```diff
@@ -254,39 +254,14 @@ class SmartCard
 };
 
 /** Reader provides card reader information, status and gives access to the smart card in it. */
-class Reader
+struct Reader
 {
-public:
-    enum class Status {
-        UNAWARE,
-        IGNORE,
-        CHANGED,
-        UNKNOWN,
-        UNAVAILABLE,
-        EMPTY,
-        PRESENT,
-        ATRMATCH,
-        EXCLUSIVE,
-        INUSE,
-        MUTE,
-        UNPOWERED,
-        _
-    };
-
-    Reader(ContextPtr context, string_t name, byte_vector cardAtr, flag_set<Status> status);
```

Comment:
We should not expose `ctx` which is inherently private, so I suggest making it private again and restoring the constructor:

```c++
    Reader(ContextPtr context, string_t readerName, byte_vector atr, bool isInserted) :
        name(readerName), cardAtr(atr), isCardInserted(isInserted), ctx(context) {}
```